README.md

```diff
@@ -4,21 +4,105 @@
 ## What is ocdev?
 OpenShift Command line for Developers
 
+## Pre-requisites
+To use ocdev you need access to OpenShift instance and OpenShift CLI installed on your local machine (`oc` command in your $PATH).
```

Comment:
s/OpenShift CLI/OpenShift client

`oc` command should be preset in $PATH

Reply:
Maybe we should also add information about the compatible version of OC ?

Reply:
> s/OpenShift CLI/OpenShift client

I've used CLI because the same term is used in official OpenShift documentation 
https://docs.openshift.org/latest/cli_reference/index.html

Reply:
I've added information about OpenShift version